boefjes/Dockerfile

```diff
@@ -22,7 +22,7 @@ ENV PATH=/home/nonroot/.local/bin:${PATH}
 ARG ENVIRONMENT
 
 COPY --from=boefjes-requirements /tmp/boefjes-requirements.txt /tmp/boefjes-requirements.txt
-COPY boefjes/requirements-dev.txt .
+COPY boefjes/requirements-dev.txt boefjes/requirements.txt .
```

Comment:
boefjes/requirements.txt is already part of /tmp/boefjes-requirements.txt. Also the COPY here isn't very useful if we don't do something with this later?

Reply:
I added a `-r requirements.txt` in the `requirements-dev.txt` to be more consistent with the other services, hence the extra COPY